tests/console/consoletest_finish.pm

```diff
@@ -58,7 +58,12 @@ sub run() {
         }
         wait_idle;
         mouse_hide(1);
-        assert_screen 'generic-desktop';
+        if (check_var("DESKTOP", "awesome")) {
+            assert_screen 'displaymanager', 200;
```

Comment:
Can you explain why `assert_screen 'generic-desktop';` does not work for awesome tests? the needles look the same except for the tags, maybe?